### PR TITLE
feat: added shopping cart page and related style

### PR DESCRIPTION
**Pull request by therajatg**
cart

### CODE REVIEW

File: docs/cart.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                    <h2>₹ 4999</h2>
+                    <div>
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                    </div>
+                </div>
+                <div>
+                    Delivery by Mon Feb 28
+                </div>
+            </div>
+
```
Comment (agrawalshanu708) [inline docs/cart.html:75]: add class to them

File: styles/cart.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    align-self: center;
+}
+
+.ecom-horizontal-card{
+    justify-content: space-between;
+}
+
+.price-table{
+    box-shadow: 5px 5px 10px grey; 
+}
```
Comment (agrawalshanu708) [inline styles/cart.css:59]: use variable

**Review by agrawalshanu708 — COMMENTED** [review]
else is looking good

File: styles/cart.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+.ecom-horizontal-card-text{
+    word-wrap: break-word;
+    overflow: auto;
+    margin-left: 1rem;
+}
+
+/* .list-horizontal{
+    justify-content: flex-start;
+    width: auto;
+} */
```
Comment (sadathshariff) [inline styles/cart.css]: remove the commented code here

**Review by sadathshariff — COMMENTED** [review]
The code looks clean but i would suggest you to give some padding and reduce the gap between rating starts

File: styles/cart.css (hunk 2 above)
Comment (therajatg) [reply styles/cart.css:59]: Good suggestion. Used variable and a class for the same.

File: docs/cart.html (hunk 1 above)
Comment (therajatg) [reply docs/cart.html:75]: I don't think it's required

File: styles/cart.css (hunk 3 above)
Comment (therajatg) [reply styles/cart.css]: done

**Review by sadathshariff — APPROVED** [review]

**Review by agrawalshanu708 — APPROVED** [review]
ohkay

**Pull request merged**